python/cudf/cudf/tests/test_parquet.py

```diff
@@ -469,6 +470,22 @@ def test_multifile_warning(datadir):
         assert_eq(expect, got)
 
 
+# Validates the metadata return path of the parquet writer
+def test_parquet_writer_return_metadata(tmpdir, simple_gdf):
+    gdf_fname = tmpdir.join("data1.parquet")
+
+    # Write out the gdf using the GPU accelerated writer
+    df_metadata = simple_gdf.to_parquet(
+        gdf_fname.strpath, index=None,
+        metadata_file_path="test/data1.parquet"
+    )
+    df_metadata_list = [ df_metadata ]
+    merged_metadata = merge_parquet_filemetadata(df_metadata_list)
+
+    # Verify that we got a valid parquet signature in the final metadata blob
+    assert merged_metadata[0:4] == ['P','A','R','1']
```

Comment:
Can we do this if `merge_parquet_filemetadata` is returning a numpy array? Maybe: `assert merged_metadata.tobytes()[0:4] == b'PAR1'`

Reply:
@rjzamora this turned out to be exactly what was needed! this passes now.